DOCKER.md

```diff
@@ -0,0 +1,123 @@
+These instructions will walk you through installing a compatible version.
+
+To install Docker, see the appropriate section below:
+[macOS](#macos), [Ubuntu](#ubuntu), [Windows](#windows)
+or consult your package management for your OS.
+
+### macOS
+
+1. Install Docker itself:
+```shell
+brew install docker
+```
+
+1. Install Docker Compose:
+    1. ```shell
+       export DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+       mkdir -p $DOCKER_CONFIG/cli-plugins
+       curl -SL https://github.com/docker/compose/releases/download/v2.27.0/docker-compose-darwin-$(uname -m) -o $DOCKER_CONFIG/cli-plugins/docker-compose
+       chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
+       ```
+    1. You can inspect that the version is appropriate by running `docker compose version` and inspecting the result.
+
+1. Install Colima as a container runtime:
+```shell
+brew install colima
+```
+
+1. Start the Colima service (and have it start on login)
+```shell
+brew services start colima
+```
+
+1. Verify that Docker works
+```shell
+docker run hello-world
+```
+
+### Ubuntu
+
+1. Open a terminal.
+
+1. Install Docker via `apt`:
+
+```shell
+sudo apt update && sudo apt install docker.io
+```
+
+1. Enable and start the Docker service
+```shell
+sudo systemctl enable docker
+sudo systemctl start docker
+```
+
+1. Give your account privileges to run Docker
+```shell
+sudo usermod -aG docker ${USER}
+```
+
+1. Verify that Docker works without root (You may need to restart your terminal session.)
+```shell
+docker run hello-world
+```
+
+1. Install Docker Compose:
+    1. ```shell
+       export DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+       mkdir -p $DOCKER_CONFIG/cli-plugins
+       curl -SL https://github.com/docker/compose/releases/download/v2.27.0/docker-compose-linux-$(uname -m) -o $DOCKER_CONFIG/cli-plugins/docker-compose
+       chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
+       ```
+    1. You can inspect that the version is appropriate by running `docker compose version` and inspecting the result.
+
+### Windows
+
+Docker on Windows is facilitated with the Docker Desktop application which you can find [here](https://www.docker.com/products/docker-desktop/).
+
+When you install that, you will need to follow the instructions in that app to enable Docker, which may require updating some system settings. There are other instructions found [here](https://docs.docker.com/desktop/install/windows-install/) that may help.
+
+1. Install Docker Desktop from [here](https://www.docker.com/products/docker-desktop/). Using instructions found [here](https://docs.docker.com/desktop/install/windows-install/).
+
+1. Start Docker Desktop. It will say "Engine running" in the lower-left corner of the Docker Desktop window.
+
+1. Start your WSL Ubuntu session.
+
+1. Verify Docker works:
+```shell
+docker run hello-world
+```
+
+1. Verify Docker Compose version is at least 2.23.
+```shell
+docker compose version
+```
+
+## Running a Native Dashboard Server
```

Comment:
I'd suggest moving this section to the top - put the "Daily reference" details above the fold, and bury the first-time setup details a little bit.